File: selenium/python/example-pytest/support/StepObjects/CommonSteps.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.ie.webdriver import WebDriver
from selenium.webdriver.support.wait import WebDriverWait

from support.PageObjects.CommonPage import CommonPage

class CommonSteps:

    def __init__(self, webdriver):
        self.driver = webdriver

    def sendForm(self, login, password):
        self.driver.find_element(By.NAME, CommonPage.nameLogin).send_keys(login)
        self.driver.find_element(By.NAME, CommonPage.namePassword).send_keys(password)
        self.driver.find_element(By.ID, CommonPage.idButtonLogin).click()

File: selenium/java/example-junit/src/test/java/support/StepObjects/CommonSteps.java
package support.StepObjects;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.Wait;
import org.openqa.selenium.support.ui.WebDriverWait;
import support.PageObjects.CommonPage;

public class CommonSteps {
    public final WebDriver driver;

    public CommonSteps(WebDriver webdriver){
        driver = webdriver;
    }

    public void sendForm(String login, String password){
        driver.findElement(By.name(CommonPage.nameLogin)).sendKeys(login);
        driver.findElement(By.name(CommonPage.namePassword)).sendKeys(password);
        driver.findElement(By.id(CommonPage.idButtonLogin)).click();
    }

}
